mscore/CMakeLists.txt

```diff
@@ -1071,6 +1072,8 @@ if (APPLE)
       ../fonts/mscoreTab.ttf
       ../fonts/mscore-BC.ttf
       ../fonts/bravura/BravuraText.otf
+      ../fonts/petaluma/PetalumaText.otf
+      ../fonts/petaluma/PetalumaScript.otf
```

Comment:
Here Petaluma Script is needed, in the same as FreeSerif is, Question is whether Petaluma Text is needed here?